app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+
+const{ getTopics } = require('./controllers/topics.controller')
+
+app.get('/api/topics', getTopics);
+
+
+app.all('*', (req,res)=>{
+    console.log("caught error")
+    res.status(400).send({ msg: 'Bad Request'})
+    })
```

Comment:
Good work implementing this custom error handler. I don't think you need the `console.log` here as you'll be sending the error response anyway.